cutlass/nvf_cutlass.h

```diff
@@ -0,0 +1,28 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <torch/torch.h>
+
+namespace nvfuser::cutlass_kernels {
+
+void nvfp4_scaled_mm(
+    torch::Tensor& output,
+    const torch::Tensor& a,
+    const torch::Tensor& b,
+    const torch::Tensor& scales_a,
+    const torch::Tensor& scales_b,
+    const torch::Tensor& alpha);
```

Comment:
I thought the cutlass kernel supported nvfp4 outputs?

Reply:
Not required for demo.